templates/patterns/accordion/pattern-accordion.html.twig

```diff
@@ -5,6 +5,8 @@
  */
 #}
 {% include '@oe-bcl/accordion' with {
+  'title': title,
+  'title_tag': title_tag|default('h2'),
```

Comment:
These defaults are already present in the BCL templates

Reply:
done.